src/main/java/no/nav/fo/veilarboppfolging/kafka/OppfolgingStatusKafkaProducer.java

```diff
@@ -30,44 +31,53 @@
 
     private final KafkaProducer<String, String> kafka;
     private final OppfolgingFeedRepository repository;
-    private final AktorService aktorService;
     private final String topicName;
 
+
+    @Value
+    @Builder
+    public static class Melding {
+        String aktoerid;
+        String veileder;
+        boolean oppfolging;
+        Boolean nyForVeileder;
+        Boolean manuell;
+        ZonedDateTime startDato;
+        Hendelse hendelse;
+    }
+
+    enum Hendelse {
+        OPPFOLGING_STARTET,
+        OPPFOLGING_AVSLUTTET,
+        ENDRING_I_MANUELL_STATUS,
+        VEILEDER_TILORDNET,
+        AKTVITETSPLAN_LEST_AV_VEILEDER
```

Comment:
hva med enhet endret?

Reply:
Det vet ikke oppfolging noe om, vi får det fra db-linken til arena

Reply:
https://github.com/navikt/veilarboppfolging/blob/dev/src/main/java/no/nav/fo/veilarboppfolging/services/OppfolgingsenhetEndringService.java

Reply:
Det kommer fra veilarbarena som poller den db-linken, tror det er bedre om vi heller får arena til å lage en kafka-topic for oss slik at vi går rett til kilden

Reply:
tänkte att det var ganska lavt terskel att legge det til att siden vi har den datan ( i mellomtiden vi venter på få den topic:en). Men liker det du har gjort hittils